testing/scenario_app/ios/Scenarios/ScenariosUITests/LocalizationInitializationTest.m

```diff
@@ -25,11 +25,16 @@ - (void)setUp {
 - (void)testNoLocalePrepend {
   NSTimeInterval timeout = 10.0;
 
+  // The locales recieved by dart:ui are exposed onBeginFrame via semantics label.
+  // There should only be one locale, as we have removed the locale prepend on iOS.
```

Comment:
Maybe tweak the comment a little to describe why there should only be one locale. A newer Flutter developer reading this might get confused about what the "locale prepend on iOS" is since it never existed before [the PR where it was removed](https://github.com/flutter/engine/pull/18379/files#diff-8cfd502d64b05500703bbedd3fea32afL765).